src/main/java/uk/gov/pay/directdebit/tasks/services/ExpireService.java

```diff
@@ -16,16 +20,23 @@
 
     private TransactionService transactionService;
     private PaymentStatesGraph paymentStatesGraph;
+    private MandateStatesGraph mandateStatesGraph;
     private static final Logger LOGGER = PayLoggerFactory.getLogger(ExpireService.class);
     private final long MIN_EXPIRY_AGE_MINUTES = 90l;
     private final PaymentState PAYMENT_EXPIRY_CUTOFF_STATUS = PaymentState.PENDING;
-    
+    private final MandateState MANDATE_EXPIRY_CUTOFF_STATUS = MandateState.PENDING;
+    private MandateService mandateService;
+
+
     @Inject
-    public ExpireService(TransactionService transactionService, PaymentStatesGraph paymentStatesGraph) {
+    ExpireService(TransactionService transactionService, MandateStatesGraph mandateStatesGraph,
```

Comment:
The constructor is not public. I don't see why it needs to be - the PR build is green - but thought I'd flag it up. I'm ambivalent on this.

Reply:
Ok thanks, I didn't think it needed to be and it builds correctly so I went with the lesser visibility.